odoo/addons/base/tests/test_expression.py

```diff
@@ -106,6 +106,17 @@ def test(op, value, expected):
         test('not ilike', 'B', ['0', 'a'])
         test('not like', 'AB', ['0', 'a', 'b', 'a b'])
 
+    def test_09_hierarchy_filtered_domain(self):
+        Partner = self.env['res.partner']
+
+        Partner = self.env['res.partner']
```

Comment:
Once is enough
```suggestion
        Partner = self.env['res.partner']
```

Reply:
Indeed... :/ some code cleaning is missing... :)